src/main/resources/swagger/api-docs.yaml

```diff
@@ -1238,10 +1292,10 @@ paths:
           content: { }
         403:
           description: Caller does not have permission to create a pet managed identity in the given managed resource group
-          content: { }
-        404:
```

Comment:
removed 404 as a response because it does not really make sense. 404 should only be used when something in the resource path cannot be found. Plus it leaks information about what exists and what doesn't